vars/discordtemplates.groovy

```diff
@@ -0,0 +1,41 @@
+def sendSuccess(String webhookUrl) {
+    discord.sendEmbed(
+        webhookUrl,
+        ':white_check_mark: BUILD SUCCEEDED :white_check_mark:',
+        '',
```

Comment:
Might want to set discord.sendEmbeds description argument to null, saves you from sending empty string parameter?